04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,19 @@
 #write your code here
+def translate(str)
+    words = str.split
+    pig_latin_words = words.map do |word|
+      if word[0] =~ /[aeiouAEIOU]/
```

Comment:
You can simplify the regex with the case-insensitive flag `i`:

```suggestion
      if word[0] =~ /[aeiou]/i
```